src/ngraph/runtime/cpu/pass/cpu_assignment.cpp

```diff
@@ -468,6 +468,8 @@ namespace ngraph
                         auto op_annotations =
                             std::make_shared<ngraph::runtime::cpu::CPUOpAnnotations>();
                         op_annotations->set_mkldnn_op(true);
+                        std::map<size_t, size_t> oi_pairs = {{0, 0}};
+                        op_annotations->set_in_place_oi_pairs(oi_pairs);
```

Comment:
is it `in_place` or `inplace`. Please be consistent throughout. (hint: It is `inplace`)